FlowCrypt/Functionality/Services/Local Contacts Service/LocalContactsProvider.swift

```diff
@@ -121,14 +120,17 @@ extension LocalContactsProvider: LocalContactsProviderType {
     func removePubKey(with fingerprint: String, for email: String) throws {
         let storage = try storage
 
-        try find(with: email)?
+        let keys = try find(with: email)?
             .pubKeys
             .filter { $0.primaryFingerprint == fingerprint }
-            .forEach { key in
-                try storage.write {
-                    storage.delete(key)
-                }
+        guard let keys = keys else {
+            return
+        }
+        for key in keys {
```

Comment:
A little bit closer to original code, and also shorter, could be `for key in keys ?? [] {` (assuming `[]` is empty array in swift). That's what I do in TypeScript. But either is fine.

Reply:
We can't do like above for this kind because keys is not normal array.
I think better to add `guard` for this case.

![image](https://user-images.githubusercontent.com/77344191/159956867-8337d907-f10c-4e92-a5c2-6f147b72ae6d.png)

Reply:
ah, got it